exammode/forms.py

```diff
@@ -0,0 +1,28 @@
+from datetime import datetime
+
+from django.forms import ModelForm, SplitDateTimeField, ModelChoiceField
+
+from .models import ExamSession
+
```

Comment:
No empty line between import statements for this project's modules. Empty lines are used between Python native modules, third-party modules (Django) and this project's modules.

Reply:
Fixed